test/unit/lib/plugins/aws/deploy/lib/uploadArtifacts.test.js

```diff
@@ -334,7 +328,8 @@ describe('uploadArtifacts', () => {
       // There were observed race conditions (mostly in Node.js v6) where this temporary home
       // folder was cleaned before stream initialized fully, hence throwing uncaught
       // ENOENT exception into the air.
-      sinon.stub(fs, 'createReadStream').returns({ path: customResourcesFilePath, on: () => {} });
+      // TODO: Can't get the mock to work
```

Comment:
I don't feel confident with introducing change to a sensitive piece of logic without sufficient test coverage or even removing existing assertions. Additionally, these tests are obsolete and before introducing any changes, it would be better to first migrate them to `runServerless` approach (as explained in https://github.com/serverless/serverless/blob/master/test/README.md) or at least introduce tests for the changed piece of codebase with `runServerless` approach.

Reply:
Sadly I think this is a trivial change that suffers from the tests being too intrusive.